src/plugins/list/schema.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+import { Block } from 'slate'
+
+function normalizeListItem(change, error) {
+    const { child, code } = error
+
+    switch (code) {
+    case 'child_type_invalid': {
+        if (child.object === 'text') {
+            change.wrapNodeByKey(child.key, Block.create('paragraph'))
```

Comment:
You're assuming `paragraph` is a valid child, but also allow configuring the plugin such that it isn't.

You can also write this as `change.wrapNodeByKey(child.key, 'paragraph')`.

Reply:
> You can also write this as change.wrapNodeByKey(child.key, 'paragraph').

This would cause: `Error: `Node.create` only accepts objects or nodes but you passed it: paragraph`, but I'l change it to `{ object: 'block', type: 'paragraph' }`

Reply:
Ah right, `wrapNodeByKey` is just a wrapper around either `wrapBlockByKey` or `wrapInlineByKey`, and thus can't distinguish between those cases if it's provided a string. `change.wrapBlockByKey(child.key, 'paragraph')` is the correct short version.

Reply:
> You're assuming paragraph is a valid child, but also allow configuring the plugin such that it isn't.

Turns out slate-edit-list on which this plugin is based already assumes that `paragraph` exists, so I retract this concern.